import_export_batches/models.py

```diff
@@ -3042,73 +3061,45 @@ def import_candidates_from_endorsement_json(self, batch_uri='', batch_set_id='',
-                if election_results['success']:
-                    election_list = election_results['election_list']
-                    if len(election_list) == 1:
-                        [election] = election_list
-                        election_name = election.election_name
-                        google_civic_election_id = election.google_civic_election_id
-                    else:
-                        # use state_code & election_date for lookup. If multiple entries found, do not set
-                        # google_civic_election_id
-                        election_results = election_manager.retrieve_elections_by_state_and_election_date(state_code,
-                                                                                                          election_day)
-                        if election_results['success']:
-                            election_list = election_results['election_list']
-                            if len(election_list) == 1:
-                                [election] = election_list
-                                election_name = election.election_name
-                                google_civic_election_id = election.google_civic_election_id
-
             if first_line:
                 # create batch_header and batch_header_map for candidate_positions
                 first_line = False
                 try:
                     batch_header = BatchHeader.objects.create(
-                        batch_header_column_000='id',
-                        batch_header_column_001='name',
-                        batch_header_column_002='twitter_url',
-                        batch_header_column_003='facebook_url',
-                        batch_header_column_004='more_info_url',
-                        batch_header_column_005='state_code',
-                        batch_header_column_006='profile_image_url',
-                        batch_header_column_007='profile_image_url_https',
-                        batch_header_column_008='candidate_ocd_division_id',
-                        batch_header_column_009='party',
-                        batch_header_column_010='election_day',
-                        batch_header_column_011='google_civic_election_id',
+                        batch_header_column_000='name',
+                        batch_header_column_001='twitter_url',
+                        batch_header_column_002='facebook_url',
+                        batch_header_column_003='more_info_url',
+                        batch_header_column_004='state_code',
+                        batch_header_column_005='office_name',
+                        batch_header_column_006='profile_image_url_https',
+                        batch_header_column_007='party',
+                        batch_header_column_008='election_day',
+                        batch_header_column_009='google_civic_election_id',
```

Comment:
We should also accept `candidate_ocd_division_id`.

Reply:
Sure @DaleMcGrew , I will rollback all changes as per your review comments. Thanks!